src/components/voting/voting.css

```diff
@@ -69,9 +72,18 @@
     overflow: auto !important;
     z-index: 9;
 }
+.info {
+    border-top: 1px solid rgba(0,0,0,0.12);
+    border-bottom: 1px solid rgba(0,0,0,0.12);
+    margin: 10px -24px 20px;
+}
+.autoCompleteTile{
+    margin-bottom: 5px;
+}
 /* react toolbar overwroght */
 .input{
-    margin-top: -15px;
+    margin-top: -10px;
+    padding-bottom: 5px;
```

Comment:
IMO it looks better without this `padding-bottom` override. (I like more the default of 20px)